### ISSUE TITLE
Issues upgrading from 9.99 to 10.34.3

### ISSUE THREAD

**Issue by IGZgustavomarin** [renovateapp/config-help#5]
I'm upgrading to 10.34.3 and I got lots of onboarding PRs even with my config set to `onboarding: false`. And also getting no PR for upgrades. I'd prefer not to have a `renovate.json` per repo.

Here is my config: 
```
module.exports = {
  extends: [
    'config:base'
  ],
  platform: 'gitlab',
  endpoint: '__MY_REPO_ENDPOINT__',
  onboarding: false,
  assignees: [ 'gustavo' ], // user_id 8: gustavo
  logLevel: 'debug',
  labels: ['upgrade'],
  automerge: true,
  docker: {
    enabled: false
  },
  meteor: {
    enabled: false
  },
  lockFileMaintenance: {
    enabled: false
  },
  groupName: 'all',
  ignoreDeps: ['cucumber', 'restify', 'mocha', 'nock'],
  repositories: [
    // Service repositories
    'services/s1',
    'services/s2',
    //...
  ]
}

```

This config was ok for 9.99, and I've checked with latest config doc. I don't know why it is creating an onboarding PR. 

Am I doing something wrong?

PS: Also, I tried lots of ways to assign PRs to a user without success. This stop working at some point during 9.x update, maybe after migrating to Gitlab API v4.

**Comment by rarkins** [comment]
If Renovate thinks the repo needs onboarding then it won’t raise upgrade PRs until after onboarding is complete. It’s possible that the `onboarding` flag is being missed so I will look at that first. 

For assigning, do you mean the onboarding PRs or upgrade PRs?

**Comment by rarkins** [comment]
Sorry, or did you mean assigning PRs in 9.x?

**Comment by IGZgustavomarin** [comment]
I've done two thing.
 - First I checked that `.npmrc` is in place and accesible (I have private packages) and maybe this changed the onboarding behaviour.
- Second, I removed the `automerge` flag.

Now PRs are being generated on each repo, and I closed all the onboarding PRs.  

Sorry the confusion about assigning PRs. It stopped working at some point on 9.x, for all PRs. But after I tried those changes above, and with version 10.34.3, its working fine, you might have addressed that at some point.

**Comment by rarkins** [comment]
No problem at all. I know Renovate has caused its share of problems for you too :)

Little strange re: onboarding. Npmrc shouldn’t impact it! If problems come back then be sure to reopen. 

GitLab’s v3 to v4 transition was painful. I tried to migrate it gracefully but got hit by some undocumented issues. In renovate v10 we dropped support for v3 completely and had multiple users test it so hopefully things are resolved or at least greatly improved!
